css/selectors/-ms-value.json

```diff
@@ -0,0 +1,69 @@
+{
+  "css": {
+    "selectors": {
+      "-ms-value": {
+        "__compat": {
+          "description": "<code>::-ms-value</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::-ms-value",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": [
+              {
+                "version_added": "16"
+              },
+              {
+                "version_added": "12",
+                "version_removed": "16",
```

Comment:
Should `version_removed ` be `15` or `16`?

Edge `15` is the final version that _does_ have the bug. The bug is fixed in Edge `16`.

Semantically, it feels like it should be `16`, because that's the version in which the bug was removed.

However, the output of `npm run render 'css.selectors.-ms-value' 1 false` seems to indicate that `version_removed` should be `15`:

<div class="htab">
    <a id="AutoCompatibilityTable" name="AutoCompatibilityTable"></a>
    <ul>
    <li class="selected">
    <a href="javascript:;">Desktop</a>
    </li>
    <li>
    <a href="javascript:;">Mobile</a>
    </li>
    </ul>
    </div><div id="compat-desktop"><table class="compat-table"><thead><tr><th>Feature</th><th>Chrome</th><th>Edge</th><th>Firefox</th><th>Internet Explorer</th><th>Opera</th><th>Safari</th></tr></thead><tbody><tr><td><a href="https://developer.mozilla.org/docs/Web/CSS/::-ms-value"><code>::-ms-value</code></a></td><td class="no-support"><span title="No support">No</span></td><td class="full-support"><p>16</p><p>12&nbsp;— 15 *</p></td><td class="no-support"><span title="No support">No</span></td><td class="full-support">10 *</td><td class="no-support"><span title="No support">No</span></td><td class="no-support"><span title="No support">No</span></td></tr></tbody></table></div><div id="compat-mobile"><table class="compat-table"><thead><tr><th>Feature</th><th>Android</th><th>Chrome for Android</th><th>Edge mobile</th><th>Firefox for Android</th><th>Opera Android</th><th>iOS Safari</th></tr></thead><tbody><tr><td><a href="https://developer.mozilla.org/docs/Web/CSS/::-ms-value"><code>::-ms-value</code></a></td><td class="no-support"><span title="No support">No</span></td><td class="no-support"><span title="No support">No</span></td><td class="no-support"><span title="No support">No</span></td><td class="no-support"><span title="No support">No</span></td><td class="no-support"><span title="No support">No</span></td><td class="no-support"><span title="No support">No</span></td></tr></tbody></table></div>

Reply:
We should add into the data what is semantically correct.
You are right that the output is wrong. I hope to fix that soon. See https://bugzilla.mozilla.org/show_bug.cgi?id=1417902